### PR TITLE
Issue/244

### PR DESCRIPTION
**Pull request by matzuk**
Tutorials module

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -14,5 +14,7 @@ include(
     ":samples:adbserver-sample",
     ":samples:kaspresso-sample",
     ":samples:kautomator-sample",
-    ":samples:kautomator-sample-app-upgrade"
+    ":samples:kautomator-sample-app-upgrade",
+
+    ":tutorials"
```
Comment (RuslanMingaliev) [inline settings.gradle.kts:19]: `tutorial` or `tutorials`? As I remember, this module should contain only a test app with the most common complex cases

File: tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
```
Comment (RuslanMingaliev) [inline tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/ExampleInstrumentedTest.kt:12]:
```suggestion
 * Instrumented test, which will be executed on an Android device.
```

File: tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/test/SimpleTest.kt [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import com.kaspersky.kaspresso.tutorials.MainActivity
+import com.kaspersky.kaspresso.tutorials.R
+import com.kaspersky.kaspresso.tutorials.screen.MainScreen
+import com.kaspersky.kaspresso.tutorials.screen.SimpleScreen
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * In this example you can observe a test tuned by default Kaspresso configuration.
```
Comment (RuslanMingaliev) [inline tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/test/SimpleTest.kt:15]:
```suggestion
 * In this example you can see a test, tuned by default Kaspresso configuration.
```

File: tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/test/SimpleTest.kt [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ * When you start the test you can see output of default Kaspresso interceptors:
+ * - a lot of useful logs
+ * - failure handling
+ * - screenshots in the device
+ * * Also you can observe the test dsl simplifying a writing of any test
```
Comment (RuslanMingaliev) [inline tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/test/SimpleTest.kt:20]:
```suggestion
 * * Also you can see the test DSL simplifying the writing of any test
```

**Review by RuslanMingaliev — DISMISSED** [review]

File: settings.gradle.kts (hunk 1 above)
Comment (matzuk) [reply settings.gradle.kts:19]: ```tutorial``` sure, fixed

File: tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/ExampleInstrumentedTest.kt (hunk 2 above)
Comment (matzuk) [reply tutorials/src/androidTest/java/com/kaspersky/kaspresso/tutorials/ExampleInstrumentedTest.kt:12]: Interesting, I removed this file earlier. Will check it.

**Review by RuslanMingaliev — APPROVED** [review]

**Review by eakurnikov — APPROVED** [review]

**Pull request merged**